app/main.py

```diff
@@ -1,6 +1,33 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+
+load_dotenv()
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "https://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    request = (
+        requests
+        .get(
+            f"{BASE_URL}/current.json?q={FILTERING}&key={API_KEY}"
```

Comment:
There is a syntax error in the f-string. Ensure that the f-string is correctly formatted with proper use of curly braces `{}` for variable interpolation.